test/unit/org/apache/cassandra/index/sai/cql/types/AsciiTest.java

```diff
@@ -26,6 +26,6 @@ public class AsciiTest extends IndexingTypeSupport
     @Parameterized.Parameters(name = "dataset={0},wide={1},scenario={2}")
```

Comment:
Aside: At some point (if it doesn't complicate future work) it might be nice to throw all these in their own package, something like `cql.types.scalar` so you don't randomly have `DataSet` and `IndexingTypeSupport` in the same package :)

Reply:
Why would we not want `DataSet` and `IndexingTypeSupport` in the same package? They are both only used by the `types` package.

Reply:
I mean you'd have `DataSet` and `IndexingTypeSupport` still in the `types` package, but all the "scalar" tests in a `types.scalar` package (ex. `AsciiTest`) just like collection tests are in the `types.collections` package. Does that make more sense?